Modules/ThirdParty/OpenJPEG/src/openjpeg/opj_includes.h

```diff
@@ -92,7 +92,7 @@ Most compilers implement their own version of this keyword ...
 
 /* MSVC 64bits doesn't support _asm */
 #if !defined(_WIN64)
-static INLINE long lrintf(float f){
+static INLINE long opj_lrintf(float x){
```

Comment:
I don't think we should change the name of the parameter, as it is referenced by the assembly code. It should stay `f`.

Reply:
Good catch. I'll issue a change

Reply:
Update has been pushed

Reply:
I saw that, I am building locally.